packages/react/scripts/build-components.js

```diff
@@ -27,7 +27,7 @@ const outputDir = path.resolve(__dirname, '..', 'dist', 'transpiled');
 
 logger.log('💅 Transpiling components...');
 execSync(
-  `npx tsc --project tsconfig.lib.json --declarationDir ${outputDir} --emitDeclarationOnly false --outDir ${outputDir}`,
+  `npx tsc --project ${path.resolve(path.join(__dirname, '..', 'tsconfig.lib.json'))} --declarationDir ${outputDir} --emitDeclarationOnly false --outDir ${outputDir}`,
```

Comment:
Right now, the script can only be run from the `packages/react`, if we try to run from `packages/react/scripts`, it fails.

```bash
Error: Command failed: npx tsc --project tsconfig.lib.json --declarationDir /Users/brion/Desktop/wso2/oxygen-ui/packages/react/dist/transpiled --emitDeclarationOnly false --outDir /Users/brion/Desktop/wso2/oxygen-ui/packages/react/dist/transpiled
    at checkExecSyncError (node:child_process:871:11)
    at execSync (node:child_process:943:15)
    at Object.<anonymous> (/Users/brion/Desktop/wso2/oxygen-ui/packages/react/scripts/build-components.js:29:1)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  status: 1,
  signal: null,
  output: [
    null,
    Buffer(70) [Uint8Array] [
      101, 114, 114, 111, 114,  32,  84,  83,  53,  48,  53,
       56,  58,  32,  84, 104, 101,  32, 115, 112, 101,  99,
      105, 102, 105, 101, 100,  32, 112,  97, 116, 104,  32,
      100, 111, 101, 115,  32, 110, 111, 116,  32, 101, 120,
      105, 115, 116,  58,  32,  39, 116, 115,  99, 111, 110,
      102, 105, 103,  46, 108, 105,  98,  46, 106, 115, 111,
      110,  39,  46,  10
    ],
    Buffer(0) [Uint8Array] []
  ],
  pid: 62188,
  stdout: Buffer(70) [Uint8Array] [
    101, 114, 114, 111, 114,  32,  84,  83,  53,  48,  53,
     56,  58,  32,  84, 104, 101,  32, 115, 112, 101,  99,
    105, 102, 105, 101, 100,  32, 112,  97, 116, 104,  32,
    100, 111, 101, 115,  32, 110, 111, 116,  32, 101, 120,
    105, 115, 116,  58,  32,  39, 116, 115,  99, 111, 110,
    102, 105, 103,  46, 108, 105,  98,  46, 106, 115, 111,
    110,  39,  46,  10
  ],
  stderr: Buffer(0) [Uint8Array] []
}
```